cylc/flow/task_job_mgr.py

```diff
@@ -753,16 +753,6 @@ def _manip_task_jobs_callback(
             or (ctx.ret_code and ctx.ret_code != 255)
         ):
             LOG.error(ctx)
-        # A polling task lets us know that a task has failed because it's
-        # log folder has been deleted whilst the task was active:
-        if (
-            getattr(ctx, 'out', None)
-            and JOB_FILES_REMOVED_MESSAGE in ctx.out
-        ):
-            LOG.error(
-                f'Task {ctx.cmd[-1]} failed because task log directory'
-                f'\n{"/".join(ctx.cmd[-2:])}\nhas been removed.'
-            )
```

Comment:
I have moved this error logging into the `_poll_task_job_callback` method seeing as it only applies for poll, not kill or submit which also use this `_manip_task_jobs_callback` method